bundles/org.openhab.ui/web/src/components/widgets/system/slide-mixin.vue

```diff
@@ -0,0 +1,58 @@
+<script>
```

Comment:
The convention is to have mixins in regular .js file, not .vue files, and thus this `<script>` tag, and the closing tag, are unnecessary.
Please rename the file to `slide-mixin.js` and remove those tags.